docs/UserGuide.md

```diff
@@ -234,7 +237,7 @@ Example:
 You can key `list` to check the index of the task you wish to mark.
 </div>
 
-![UserGuide-mark](images/UserGuide-massmark-after.png)
+![Mark a task](images/Ui-MarkTask.png)
 Figure 3.6: Example of Harmonia after marking a task
```

Comment:
Should we say '... after marking task 4' to make it clearer? It may be better since the description in line 224 is very specific.